tests/ensure_provider_tests.py

```diff
@@ -21,17 +21,12 @@
           package:
             name: NetworkManager
             state: present
-        - name: Get NetworkManager package info
-          yum:
-            list: NetworkManager
-          register: networkmanager_info
+        - name: Get package info
+          package_facts:
         - name: Get NetworkManager version
           set_fact:
-            networkmanager_version: "{{ networkmanager_info.results |
-              selectattr('yumstate', 'match', '^installed$') |
-              map(attribute='version') | list | first }}"
-          when: true
-          changed_when: false
+            networkmanager_version: "{{
```

Comment:
What is the problem with the previous way of getting the networkmanager_version ?

Reply:
The use of the `yum` module.  This conflicts with the package manager used by ostree systems.  Using the `package` and `package_facts` modules works correctly on ostree systems.

Why was the `yum` module used?  Is this a performance issue?  Using `yum` directly is much, much faster than using `package_facts`?

Reply:
The time when I was implementing it, I was not aware of `package` and `package_facts` module can get the NetworkManager version so easily.